ONSdigital/eq-author-app
# Improved content picker for piping

### What is the context of this PR?
New design for content picker

### How to review 
1. Ensure piping and "select an answer" in calcuated summary page still works.
2. Ensure styling and functionality match prototype: [https://prototype-picker--author-app.netlify.com/](https://prototype-picker--author-app.netlify.com/)
3. Should comply with acceptance criteria: [Answer picker](https://trello.com/c/b9jLEjJN/1025-improve-content-picker-piping-an-answer-6) and [Metadata picker](https://trello.com/c/AwyAOkvA/1216-improve-content-picker-piping-metadata-4)
